pkg/keyctl/sys_linux.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2015 Jesse Sipprell. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build linux
+
+package keyctl
+
+import (
+	"log"
+	"syscall"
+	"unsafe"
+)
+
+type keyctlCommand int
+
+type keyID int32
+
+const (
+	keySpecSessionKeyring keyID = -3
+	keySpecUserKeyring    keyID = -4
+)
+
+const (
+	keyctlGetKeyringID keyctlCommand = 0
+	keyctlSetPerm      keyctlCommand = 5
+	keyctlLink         keyctlCommand = 8
+	keyctlUnlink       keyctlCommand = 9
+	keyctlSearch       keyctlCommand = 10
+	keyctlRead         keyctlCommand = 11
+)
+
+var debugSyscalls bool
```

Comment:
We can remove this along with its users. Looks like development artefact that can only be controlled in the code.

Reply:
Let's remove this.